doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -9,60 +9,75 @@ Import modules into IntelliJ
 ----------------------------
 
 #. Run the IntelliJ IDE
-#. Select ``File -> New -> Project from Existing Sources...``. 
-#. Navigate to the ``geoserver/src/pom.xml`` directory and click ``Open``.
+#. From the initial panel select ``Open``.
+
+   .. image:: img/intellij_open.png
+      :width: 500
+
+#. Navigate to the ``geoserver/src/pom.xml`` directory and click ``OK``.
 
    .. image:: img/intellij_import.png
-      :width: 600
+      :width: 350
 
-#. Click ``Next``, leaving all profiles unchecked.
+#. When asked click on ``Open as a Project``.
 
-   .. image:: img/intellij_import_profile.png
-      :width: 600
+   .. image:: img/intellij_project.png
+      :width: 350
 
-#. Click ``Next``, leaving the geoserver project checked.
+#. Optionally, depending on which platform you have, IntelliJ may ask you to ``Trust the Project``.
```

Comment:
Normally we write the text in the third person, so for example:

```suggestion
#. Optionally, depending on which platform you have, IntelliJ may ask to ``Trust the Project``.
```

Unfortunately, the convention is not always used consistently (even in this document, before this PR...)